forge-gui/tools/mtg-data.txt

```diff
@@ -52707,7 +52707,7 @@ White creatures you control get +0/+1.
 {R}{W}, Exile Legion's Initiative: Exile all creatures you control. At the beginning of the next combat, return those cards to the battlefield under their owner's control and those creatures gain haste until end of turn.
 DGM M
 
-Legions of Lim-Dul
+Legions of Lim-Dûl
```

Comment:
I don't think we use this file anymore. We should probably just delete it from the repo

Reply:
yeah that should be deleted

and maybe the tools folder be cleaned too